apps/src/dropletUtils.js

```diff
@@ -190,9 +190,14 @@ exports.generateDropletPalette = function (codeFunctions, dropletConfig) {
       }
```

Comment:
General note for dropletUtils.js - it seems like you should need to update the standardConfig blocks in this file to also have their titles removed (like you did for the various app-specific dropletConfig blocks)